### PR TITLE
[KYUUBI #2337] [DOCS] Access Kyuubi with Kyuubi JDBC Driver

### LINKED ISSUE #2337: [DOCS] Access Kyuubi with Kyuubi JDBC Driver
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Which parts of the documentation do you think need improvement?

Access Kyuubi with Kyuubi JDBC Documentation is missing

example 

https://kyuubi.apache.org/docs/r1.5.0-incubating/client/hive_jdbc.html

### Affects Version(s)

master

### Improving the documentation

_No response_

### Anything else

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by zwangsheng**
### _Why are the changes needed?_

Access Kyuubi with Kyuubi-Hive-JDBC Documentation is missing
Close #2337 

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by zwangsheng** [comment]
Here is the screenshots for doc
481649774905_ pic

**Comment by zwangsheng** [comment]
Please help check, @yaooqinn @pan3793 @ulysses-you

**Comment by zwangsheng** [comment]
Travis CI Test Fail, this seems to have nothing to do with PR. Retest plz.

**Comment by pan3793** [comment]
We recommend using `kyuubi-hive-jdbc-shaded` because it has less transitive dependencies, and we also need to mention user if they already have `hadoop-common` dependency in there classpath, they should exclude `hadoop-client-api` from the `kyuubi-hive-jdbc-shaded` to avoid class confliction.

**Comment by zwangsheng** [comment]
> We recommend using `kyuubi-hive-jdbc-shaded` because it has less transitive dependencies, and we also need to mention user if they already have `hadoop-common` dependency in their classpath, they should exclude `hadoop-client-api` from the `kyuubi-hive-jdbc-shaded` to avoid class confliction.

Thanks for you reminding, will fix later.

**Comment by ulysses-you** [comment]
> we also need to mention user if they already have hadoop-common dependency in their classpath, they should exclude hadoop-client-api from the kyuubi-hive-jdbc-shaded to avoid class confliction.

it sounds not easy for user ...

File: docs/client/kyuubi_jdbc.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ - limitations under the License.
+ -->
+
+<div align=center>
+
+![](../imgs/kyuubi_logo.png)
+
+</div>
+
+# Access Kyuubi with Kyuubi JDBC and ODBC Drivers
```
Comment (yaooqinn) [inline docs/client/kyuubi_jdbc.md:24]:
```suggestion
# Access Kyuubi with Kyuubi JDBC Driver
```

**Comment by pan3793** [comment]
> > we also need to mention user if they already have hadoop-common dependency in their classpath, they should exclude hadoop-client-api from the kyuubi-hive-jdbc-shaded to avoid class confliction.
> 
> it sounds not easy for user ...

Another option is make `hadoop-client-api` as optional, then use need to add there own hadoop jars into classpath

File: docs/client/kyuubi_jdbc.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+![](../imgs/kyuubi_logo.png)
+
+</div>
+
+# Access Kyuubi with Kyuubi JDBC Drivers
+
+## Install Kyuubi JDBC
+
+For programing, the easiest way to get `kyuubi-hive-jdbc-shaded` is from [the maven central](https://mvnrepository.com/artifact/org.apache.hive/hive-jdbc). For example,
```
Comment (ulysses-you) [inline docs/client/kyuubi_jdbc.md:28]: how can we get kyuubi jar from hive repo ..

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**